web_digital_sign/__openerp__.py

```diff
@@ -31,15 +31,9 @@
         This module provides the functionality to store digital signature image for a record.
         The example can be seen into the User's form view where we have added a test field under signature.
     """,
-    'data': ['users_view.xml'],
-    'js':[
-          "static/lib/excanvas.js",
-          "static/lib/jquery.signature.js",
-          "static/src/js/digital_sign.js",
-    ],
-    'css':[
```

Comment:
The js files are not necesary there on V8.0 you need to inherit the asset_frontend.

Regards.